spec/features/edit_document_associations_spec.rb

```diff
@@ -36,22 +44,28 @@ def and_i_click_on_edit_associations
 
   def then_i_can_see_the_current_selections
     @request = stub_publishing_api_put_content(Document.last.content_id, {})
-    expect(page).to have_select("associations[association_id][]",
+    expect(page).to have_select("associations[multi_association_id][]",
                                  selected: "Initial association")
+    expect(page).to have_select("associations[single_association_id][]",
+                                selected: "Selected association")
   end
 
   def when_i_edit_the_associations
-    select "Association to select 1", from: "associations[association_id][]"
-    select "Association to select 2", from: "associations[association_id][]"
-    unselect "Initial association", from: "associations[association_id][]"
+    select "Association to select 1", from: "associations[multi_association_id][]"
+    select "Association to select 2", from: "associations[multi_association_id][]"
+    unselect "Initial association", from: "associations[multi_association_id][]"
 
+    select "Other association", from: "associations[single_association_id][]"
```

Comment:
Again, this reads strangely from top to bottom, as it looks like ‘Other’ somehow relates to ‘Association to select 1’, etc.